src/kernels/jupyter/launcher/notebookStarter.node.ts

```diff
@@ -291,7 +291,7 @@ export class NotebookStarter implements INotebookStarter {
         const args: string[] = [];
         // Check for a docker situation.
         try {
-            const cgroup = await this.fs.readLocalFile('/proc/self/cgroup').catch(() => '');
+            const cgroup = await this.fs.readFile(Uri.file('/proc/self/cgroup')).catch(() => '');
```

Comment:
Whether it local or not is not a concern of the API, but its a concern of the implementation.
I.e. how we copy & read files should not be determined by the API, OOP can deal with the complications by having a custom implemementation of the IFileSystem interface.

Thus, in most places we should now be able to just use `IFileSystem` instead of `IFileSystemNode`

E.g. this class now just uses `IFileSystem` and not a custom interface.